DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: flipStandardCharts
 Type: Package
 Title: Standard R interactive charts
-Version: 1.15.6
```

Comment:
Seeing the following check NOTE, can we remove these packages?
![image](https://user-images.githubusercontent.com/5276419/97129715-b9d04e80-1793-11eb-9333-de8b701f4514.png)

Reply:
Good point. I've removed these packages and now. And ggplot2movies was alread in Suggests.

Reply:
Thanks. For `ggplot2movies` I'm was wondering why the package isn't installed on the R servers. I got an error trying to build/check the package on dev because `ggplot2movies` was not installed. If the the package isn't worth installing on our servers, why do we bother Suggest'ing it? Can it be removed?